lib/challenge.js

```diff
@@ -274,11 +307,16 @@ let nestedArrays = [
 ];
 
 function prompt13() {
+  for (i = 0; i < nestedArrays.length; i++) {
+    for (j + 0; j < nestedArrays[i].length; j++) {
+      console.log(nestedArrays[i][j])
+    }
+  }
```

Comment:
Check out line 311.  Notice anything strange?

Make sure to test your code and go through it line by line.